### PR TITLE
test: add test for creating asset dashboard

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [x] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [ ] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
Please focus on the **description**.

### CODE REVIEW

File: apps/web/tests/dashboards/create-dashboards/create-compliacne-overview-dashboard.spec.ts [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+            }
+            expect(widgetNameList).toEqual([
+                'Compliance Check Status',
+                'Total Failure and Severity',
+                'Count of Pass and Fail Findings by Region',
+                'Count of Fail Findings by Service',
+                'Trend of Pass and Fail Findings by Service',
+                'Severity Status by Service',
+            ]);
+        });
```
Comment (yuda110) [inline apps/web/tests/dashboards/create-dashboards/create-compliacne-overview-dashboard.spec.ts:72]: description: Add checking dashboard variables and widget list

File: apps/web/tests/dashboards/create-dashboards/create-compliacne-overview-dashboard.spec.ts [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+            await expect(locatorScope).toContainText(' Workspace ');
+            await expect(locatorName).toContainText(dashboardName);
+        });
+
+        await test.step('8. Delete dashboard', async () => {
+            const deleteButton = await page.locator('.dashboard-title-icon-buttons-wrapper .delete-button');
+            await deleteButton.click();
+            await page.getByRole('button', { name: 'Confirm' }).click();
+            await expect(page).toHaveURL(/dashboards\/all/);
```
Comment (yuda110) [inline apps/web/tests/dashboards/create-dashboards/create-compliacne-overview-dashboard.spec.ts:93]: description: I updated here. add 'expect'

File: apps/web/tests/dashboards/create-dashboards/create-compliacne-overview-dashboard.spec.ts [hunk 3]
```diff
@@ -0,0 +1,96 @@
+import { expect, test } from '@playwright/test';
+
+test.describe('Create Compliance Overview Dashboard', () => {
+    test('Create entire Workspace dashboard', async ({ page }) => {
+        page.goto('/');
+
+        await test.step('1. Go to Create New Dashboard page', async () => {
+            //
+            await page.locator('.gnb-menu .menu-button').first().click();
+            const createNewDashboardButton = await page.locator('.g-n-b-dashboard-menu .sub-menu').nth(1).locator('.gnb-sub-contents');
```
Comment (yuda110) [inline apps/web/tests/dashboards/create-dashboards/create-compliacne-overview-dashboard.spec.ts:10]: description: Use `locator`, not `getByText`

**Review by sulmoJ — APPROVED** [review]
It feels like I have a much better understanding now after using a locator!

**Review by skdud4659 — APPROVED** [review]
LGTM! luvit!

**Pull request merged**